src/main/java/com/kenshoo/xlsx2csv/XlsxToCsvConverter.java

```diff
@@ -86,15 +89,24 @@ private boolean isLastLine(boolean hasNext) {
         return !hasNext;
     }
 
-    private Iterator<Row> getRowIterator(InputStream inputStream) {
+    private Optional<Iterator<Row>> getRowIterator(InputStream inputStream) {
         final Workbook workbook = StreamingReader.builder()
                 .rowCacheSize(10)
                 .bufferSize(1024)
```

Comment:
It was a recommendation by the guy who wrote the streaming api. I think it just makes the user deal with things that are under the hood, but maybe we should supply a constructor with those parameters and a default one that just uses these ones. Sounds good?